web/docs/installation/deploy-a-node/macos.md

```diff
@@ -0,0 +1,4 @@
+# macOS
+
+We currently do not offer a native package for macOS. Please use the
+[Docker](docker.md) setup in the meantime.
```

Comment:
This could also refer to the build-from-source instructions.

Reply:
I would say this is misleading due to the absence of the platform plugin. Building from source is only a developer use case.

Reply:
Ah, I conveniently forgot about that being private.